cluster_info/dodrio-gpu-rome-a100-40.json

```diff
@@ -0,0 +1,13 @@
+{
+    "qdel_cmd_tpl": "scancel --cluster=dodrio {{ cluster_job_id }}",
+    "worker_bin_path" : "__worker_path__/rocky8_icelake/cryosparc_worker/bin/cryosparcw",
```

Comment:
You will need to replace `rocky8_icelake` with something like `RHEL8_zen2-ampere-ib`.

Reply:
see 1294fcd